lib/presentation/pages/edit_current_trip_page/edit_current_trip_page.dart

```diff
@@ -58,12 +58,13 @@ class EditCurrentTripPage extends StatelessWidget {
                     const SizedBox(height: AppDimensions.d40),
                     IconList(
                       icons: transportIcons,
+                      onIconSelected: (TransportMethods value) {},
```

Comment:
```suggestion
                      onIconSelected: (TransportMethods value) {},
```
TransportMethods <- I though it is referring to some functions. We already have TransportType enum maybe you can use it?